web/src/sentry.ts

```diff
@@ -8,18 +8,26 @@ import {page_params} from "./page_params";
 type UserInfo = {
     id?: string;
     realm: string;
-    role: string;
+    role?: string;
 };
 
 if (page_params.server_sentry_dsn) {
     const url_matches = [/^\//, new RegExp("^" + _.escapeRegExp(page_params.webpack_public_path))];
     if (page_params.realm_uri !== undefined) {
         url_matches.push(new RegExp("^" + _.escapeRegExp(page_params.realm_uri) + "/"));
     }
-    const sentry_key = page_params.realm_sentry_key || "(root)";
+    const sentry_key =
+        // No parameter is the portico pages, empty string is the empty realm
+        page_params.realm_sentry_key === undefined
```

Comment:
We do support serving `page_params` to portico pages, see `DEFAULT_PAGE_PARAMS`. So we could consider instead just having `realm_sentry_key` be always present?

Reply:
Oh I see, you are using that -- just not having that logic be responsible for setting `realm_sentry_key` in these extra conditional cases.